src/platforms/ios/iostunnel.swift

```diff
@@ -100,6 +102,11 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
                     GleanMetrics.Pings.shared.daemonsession.submit(reason: .daemonStart)
                 }
 
+                // The internal DNS endpoint results is in the 10.x.x.x restricted IP range. For reasons I couldn't
+                // determine, pinging restricted IP addresses failed. Thus, using other ones.
+                let endpointIP = String(describing: tunnelConfiguration.peers.first?.endpoint?.host ?? "8.8.8.8")
```

Comment:
Why start connectionhealth if `tunnelConfiguration.peers.first?.endpoint?.host` is empty? 🤔 

I'm tending to either assert it's there, cause otherwise we should hopefully (🤞 ) have an adapterError, or we should not start the timer. Wdyt?

Reply:
Changed this to a guard with an assertion, and a note about why we don't change behavior on prod. Let me know if you think we should take a different approach.